pages/assets/[id].tsx

```diff
@@ -56,20 +56,9 @@ export default function AssetInfo() {
         <Box my="0.5rem">
           <h3>Asset History</h3>
         </Box>
-        {descriptions.loaded ? (
-          <AssetHistory assetHistory={descriptions.data} />
-        ) : (
-          <HStack justifyContent="center" py="6rem">
```

Comment:
I didn't mean to remove that spinner thing it looks nice, I mean when you load block the cards has gray skeleton.
![image](https://user-images.githubusercontent.com/39185158/221622911-b0423f3d-d5bf-45e1-9ca8-882cb40b9710.png)